includes/admin/reporting/reports.php

```diff
@@ -2548,377 +2547,454 @@ function display_export_report() {
+										'show_option_all'  => false,
+									)
+								);
+								?>
+							<label for="edd_reports_filter_regions" class="screen-reader-text"><?php esc_html_e( 'All Regions or Select Region', 'easy-digital-downloads' ); ?></label>
+								<?php
+								echo EDD()->html->select(
+									array(
+										'name'             => 'region',
+										'id'               => 'edd-reports-filter-regions',
+										'options'          => edd_get_shop_states(),
+										'chosen'           => true,
+										'selected'         => false,
+										'show_option_none' => false,
+										'placeholder'      => __( 'All Regions', 'easy-digital-downloads' ),
+										'show_option_all'  => false,
+									)
+								);
 
-									echo EDD()->html->date_field( array(
-										'id'          => 'edd-taxed-orders-export-start',
-										'class'       => 'edd-export-start',
-										'name'        => 'start',
-										'placeholder' => _x( 'From', 'date filter', 'easy-digital-downloads' )
-									) );
+								wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' );
+								?>
+							<input type="hidden" name="edd-export-class" value="EDD_Batch_Taxed_Orders_Export"/>
+							<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
+						</form>
+					</div>
+				</div>
+
+				<div class="postbox edd-export-customers">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Customers', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php printf( esc_html__( 'Download a CSV of customers. Select a taxonomy to see all the customers who purchased %s in that taxonomy.', 'easy-digital-downloads' ), edd_get_label_plural( true ) ); ?></p>
+						<form id="edd-export-customers" class="edd-export-form edd-import-export-form" method="post">
+							<?php
+							$taxonomies = get_object_taxonomies( 'download', 'names' );
+							$taxonomies = array_map( 'sanitize_text_field', $taxonomies );
+
+							$placeholders = implode( ', ', array_fill( 0, count( $taxonomies ), '%s' ) );
+
+							$taxonomy__in = $wpdb->prepare( "tt.taxonomy IN ({$placeholders})", $taxonomies );
+
+							$sql = "SELECT t.*, tt.*, tr.object_id
+									FROM {$wpdb->terms} AS t
+									INNER JOIN {$wpdb->term_taxonomy} AS tt ON t.term_id = tt.term_id
+									INNER JOIN {$wpdb->term_relationships} AS tr ON tr.term_taxonomy_id = tt.term_taxonomy_id
+									WHERE {$taxonomy__in}";
+
+							$results = $wpdb->get_results( $sql );
+
+							$taxonomies = array();
+
+							if ( $results ) {
+								foreach ( $results as $r ) {
+									$t = get_taxonomy( $r->taxonomy );
+									$taxonomies[ absint( $r->term_id ) ] = $t->labels->singular_name . ': ' . esc_html( $r->name );
+								}
+							}
+							?>
+							<label for="taxonomy" class="screen-reader-text"><?php esc_html_e( 'All Taxonomies or Select Taxonomy', 'easy-digital-downloads' ); ?></label>
```

Comment:
I would change this to be just `'Select Taxonomy'` (same for select download), as "All" is the first option in the list.

Reply:
What do you think about the Country, Region and Status ones?  I didn't realize that a screen reader would read the default choice?

Reply:
Should I do this for all of the dropdowns (Countries, Statuses...)?  I didn't realize the screen reader would read the default.

Reply:
Yes, it will read the label, and then the options. So you'd end up with "Select Taxonomy. All Taxonomies. Taxonomy 1" vs. "All Taxonomies or Select Taxonomy. All Taxonomies. Taxonomy 1".

I am not sure, but I think that the same rules will apply to both regular `select` inputs and ones implementing Chosen. But looking at these, I think they may have their own...not accessibility, but usability, which isn't really that different, issue. I'm finding that the Chosen inputs don't actually have an "all" option, at least not the ones I'm checking, so if I select a download, or a country, there is no way for me to get back to "All Downloads" or "All Countries" unless I refresh the screen. So I think maybe we need to double check that, to make sure that the "all" options are available, even after a selection is made?

So under "Export Customers", I can select a taxonomy and then go back to "all" (standard `select`), but not downloads (which uses Chosen).

But yes, I would update the labels for the select inputs, and we can dig into the "all" as well, I hope.